qa/acceptance/spec/shared_examples/updated.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../spec_helper'
+require          'logstash/version'
+
+RSpec.shared_examples "updated" do |logstash|
+
+  before (:all) { logstash.snapshot }
+  after  (:all) { logstash.restore }
+
+  it "can update on #{logstash.host}" do
+    logstash.install("5.0.0-alpha2", "./")
```

Comment:
updated this to be read from the same config file as the vagrant platforms, I guess this is the best that works to have everything DRY.... next thing is obviously to name this file properly, but this can be done later.